proto/expr.proto

```diff
@@ -120,6 +120,39 @@ message InputRefExpr {
   int32 column_idx = 1;
 }
 
+// The items which can occur in the select list of `ProjectSet` operator.
```

Comment:
Use /// instead of // for docstring

Reply:
In proto, we should use // instead of /// 😄 https://github.com/singularity-data/risingwave/pull/3699